Sources/JWTKit/Vendor/AppleIdentityToken.swift

```diff
@@ -63,10 +63,10 @@ public struct AppleIdentityToken: JWTPayload {
     }
 }
 
-extension AppleIdentityToken {
+public extension AppleIdentityToken {
     /// Taken from https://developer.apple.com/documentation/authenticationservices/asuserdetectionstatus
     /// With slight modification to make adding new cases non-breaking.
-    public struct UserDetectionStatus: OptionSet, Codable {
+    struct UserDetectionStatus: OptionSet, Codable, Sendable {
```

Comment:
I think this type needs to be public so users can see it

Reply:
It is, SwiftFormat put the `public` on the extension rather than on the struct itself